src/zc/buildout/tests.py

```diff
@@ -2408,12 +2408,12 @@ def wont_downgrade_due_to_prefer_final():
     ... parts =
     ... ''')
 
-    >>> [v] = [l.split('= >=', 1)[1].strip()
+    >>> [v] = [str(l.split('= >=', 1)[1].strip())
```

Comment:
It's really odd, now that you mention it. :/

So odd I now can't reproduce it. WTF? reverted.